### DISCUSSION TITLE
Nebula Types

### DISCUSSION THREAD

**Discussion by RConflit** [Stellarium/stellarium#3117] [General]
Hello, all. 

I'm curious about the descriptors that come after some of the nebula types in Stellarium. For example, M1, the Crab Nebula is listed as "supernova remnant (F; 2, 3, 3)." The Heart Nebula is designated as "cluster associated with nebulosity (III3pn; 3, 3, 3)." 

I understand what the words mean, but what do the parenthetical references stand for? Can't find documentation anywhere. 

Thanks, everyone.

**Comment by alex-w** [comment]
Morphological type of DSO, see morphological description near end of infobox for details ;)

**Comment by RConflit** [comment]
Thank you, Alex. That's very helpful. 

But what do the numbers, for example, "2, 3, 3" signify. The morphological description of the objects does not make that clear. 

And are you aware of a listing, somewhere, of what the letters representing the morphological descriptions mean?

Thanks, again.

**Reply by alex-w** [reply]
"2, 3, 3" = "elliptical form, filamentary structure, brightest". The "morphological description" are human readable version for morphological type of DSO (except galaxies).

**Comment by RConflit** [comment]
OK. I see that now. 

Thanks for the assist, sir!
